README.md

```diff
@@ -30,7 +30,7 @@ All import blocks use one TAB(`\t`) as Indent.
 ## Download
 
 ```shell
-$ go get github.com/daixiang0/gci
+$ go install github.com/daixiang0/gci@latest
```

Comment:
Please add a comment about install a release version.

Reply:
Is that what you had in mind?

![image](https://user-images.githubusercontent.com/10882062/182326498-076817d1-f463-4042-aa8a-520b89a6f00a.png)

Reply:
yes.